Update key-vault-ovw-storage-keys.md

eNeRGy164 opened the pull request:
Fix samples, make commands more consistent, use less .NET in PowerShell statements

azurecla commented:
Hi __@crawford__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (alex.crawford@coreos.com). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

szarkos commented:
Alex and I discussed this.  I agree this makes sense to clarify what minimum version is supported, rather than listing a couple specific alpha or beta releases.

eNeRGy164 committed:
Update key-vault-ovw-storage-keys.md

Fix samples, make commands more consistent, use less .NET in PowerShell statements

fhokholdMSFT commented on articles/key-vault/key-vault-ovw-storage-keys.md:
````diff
@@ -162,20 +157,19 @@ Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -Accoun
 ```powershell
-Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob -ResourceType Container,Service -VaultName yourtest1  -AccountName msak01 -Name blobsas1 -Protocol HttpsOnly -ValidityPeriod ([System.Timespan]::FromDays(1)) -Permission Read,List
-Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob -ResourceType Container,Service,Object -VaultName yourtest1  -AccountName msak01 -Name blobsas2 -Protocol HttpsOnly -ValidityPeriod ([System.Timespan]::FromDays(1)) -Permission Read,List,Write
+Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob -ResourceType Container,Service -VaultName yourtest1 -AccountName msak01 -Name blobsas1 -Protocol HttpsOnly -ValidityPeriod "01.00:00:00" -Permission Read,List
````
@BrucePerlerMS I prefer it the original way here Bruce I think it is more programmatic.

BrucePerlerMS replied:
Agreed. I'll reject the PR.

fhokholdMSFT commented on articles/key-vault/key-vault-ovw-storage-keys.md:
````diff
@@ -162,20 +157,19 @@ Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -Accoun
 ```powershell
-$regenPeriod = [System.Timespan]::FromDays(3)
-Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -AccountResourceId /subscriptions/subscriptionId/resourceGroups/yourresgroup1/providers/Microsoft.Storage/storageAccounts/yourtest1 -ActiveKeyName key2 -RegenerationPeriod $regenPeriod
+Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -AccountResourceId /subscriptions/subscriptionId/resourceGroups/yourresgroup1/providers/Microsoft.Storage/storageAccounts/yourtest1 -ActiveKeyName key2 -RegenerationPeriod "03.00:00:00"
````
@BrucePerlerMS I prefer it the original way here as well Bruce I think it is more programmatic. I like to minimize hard coding values and long strings

BrucePerlerMS replied:
Agreed. I'll reject the PR.
